tests/helper/component_podman.go

```diff
@@ -12,54 +14,72 @@ import (
 
 // PodmanComponent is an abstraction for a Devfile Component deployed on podman
 type PodmanComponent struct {
-	name string
-	app  string
+	componentName string
+	app           string
+	containerName string
 }
 
-func NewPodmanComponent(name string, app string) *PodmanComponent {
+func NewPodmanComponent(componentName string, app string, containerName string) *PodmanComponent {
 	return &PodmanComponent{
-		name: name,
-		app:  app,
+		componentName: componentName,
+		app:           app,
+		containerName: containerName,
 	}
 }
 
 func (o *PodmanComponent) ExpectIsDeployed() {
-	podName := fmt.Sprintf("%s-%s", o.name, o.app)
+	podName := fmt.Sprintf("%s-%s", o.componentName, o.app)
 	cmd := exec.Command("podman", "pod", "list", "--format", "{{.Name}}", "--noheading")
 	stdout, err := cmd.Output()
 	Expect(err).ToNot(HaveOccurred())
 	Expect(string(stdout)).To(ContainSubstring(podName))
 }
 
 func (o *PodmanComponent) ExpectIsNotDeployed() {
-	podName := fmt.Sprintf("%s-%s", o.name, o.app)
+	podName := fmt.Sprintf("%s-%s", o.componentName, o.app)
 	cmd := exec.Command("podman", "pod", "list", "--format", "{{.Name}}", "--noheading")
 	stdout, err := cmd.Output()
 	Expect(err).ToNot(HaveOccurred())
 	Expect(string(stdout)).ToNot(ContainSubstring(podName))
 }
 
 func (o *PodmanComponent) Exec(container string, args ...string) string {
-	containerName := fmt.Sprintf("%s-%s-%s", o.name, o.app, container)
+	containerName := fmt.Sprintf("%s-%s-%s", o.componentName, o.app, container)
 	cmdargs := []string{"exec", "--interactive"}
 	cmdargs = append(cmdargs, "--tty")
 	cmdargs = append(cmdargs, containerName)
 	cmdargs = append(cmdargs, args...)
 
 	command := exec.Command("podman", cmdargs...)
 	out, err := command.Output()
+	if err != nil {
+		if exiterr, ok := err.(*exec.ExitError); ok {
+			err = fmt.Errorf("%s: %s", err, string(exiterr.Stderr))
+		}
+	}
 	Expect(err).ToNot(HaveOccurred())
 	return string(out)
 }
 
 func (o *PodmanComponent) GetEnvVars() map[string]string {
-	podName := fmt.Sprintf("%s-%s", o.name, o.app)
-	podDef := getPodDef(podName)
-	res := map[string]string{}
-	for _, env := range podDef.Spec.Containers[0].Env {
-		res[env.Name] = env.Value
+	envs := o.Exec(o.containerName, "env")
+	fmt.Printf("\n\n%s\n\n", envs)
```

Comment:
Maybe we should void printing env vars (for security concerns, if the image contains sensitive data)? Or print to the `GinkgoWriter` to have them printed only if the current test fails (or when running Ginkgo in verbose mode)..

Reply:
This was added only for debugging. I'll remove the line. Thanks